packages/breadcrumb/stories.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import { BreadcrumbContainer, useBreadcrumb } from './src';
+
+// This is easier than injecting via webpack shenanigans
+const breadcrumbscss = (
+  <link
+    href="https://unpkg.com/@zendeskgarden/css-breadcrumbs@0.2.2/dist/index.css"
```

Comment:
Outside the scope of this PR, but we should be able to provide global CSS via an import in the `config.js` file.

Reply:
Thanks for the heads up I ended up installing it as a devDep see e26180b